test/views/agent/partials/PenaltiesTilePartialSpec.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.agent.partials
+
+import messages.partials.PenaltiesTileMessages
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import views.ViewBaseSpec
+import views.html.agent.partials.PenaltiesTilePartial
+
+class PenaltiesTilePartialSpec extends ViewBaseSpec {
+
+  val penaltiesTilePartial: PenaltiesTilePartial = inject[PenaltiesTilePartial]
+
+  "PenaltiesTilePartial view" when {
```

Comment:
Could you please select the elements by ID rather than by CSS selector in this file as it makes the tests more robust?